commands/cluster-getkeysinslot.md

```diff
@@ -1,7 +1,7 @@
-The command returns an array of keys names stored in the contacted node and
-hashing to the specified hash slot. The maximum number of keys to return
-is specified via the `count` argument, so that it is possible for the user
-of this API to batch-processing keys.
+The command returns an array of key names stored in the contacted node and
+hashing to the specified hash slot **in the currently selected database**. 
+The maximum number of keys to return is specified via the `count` argument, 
+so that it is possible for the user of this API to batch-process keys.
```

Comment:
Since you're here, can you clarify that batch processing only works if you are deleting the keys. If you are trying to scan the slot, you can't batch process it. 

Also, I don't think we need to bold it's in the currently selected database.